code-samples/creating-your-bot/event-handling/index.js

```diff
@@ -1,10 +1,20 @@
 const fs = require('node:fs');
 const path = require('node:path');
-const { Client, GatewayIntentBits } = require('discord.js');
+const { Client, Collection, Events, GatewayIntentBits } = require('discord.js');
```

Comment:
This is no longer used as the event handler has come into play.

```suggestion
const { Client, Collection, GatewayIntentBits } = require('discord.js');
```